akka-osgi/build.sbt

```diff
@@ -1,6 +1,7 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
 
 AkkaBuild.defaultSettings
+AkkaBuild.dontPublishSettings
```

Comment:
I count this as a yes. ;)

I'm no sbt expert so can't promise a quick impl. Separate ticket?